### PR TITLE
Move Redux Store into Chat. Remove DebugView.

### PR DESCRIPTION
**Pull request by eanders-MS**
Tried removing the store subscription/forceUpdate from child components. It didn't work. The control stopped responding.

### CODE REVIEW

**Comment by billba** [comment]
Oh yeah, we'd need one in chat. Let's leave it this way for now.

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

File: src/Store.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Store, Reducer, createStore, combineReducers } from 'redux';
+import { Store, Reducer, createStore as reduxCreateStore, combineReducers } from 'redux';
 import { Activity, IBotConnection, User } from './BotConnection';
 import { FormatOptions } from './Chat';
 import { strings, Strings } from './Strings';
```
Comment (billba) [inline src/Store.ts]: I propose exporting type ChatStore = Store<ChatState> - that way all the places where you set props: Store<ChatStore> can become props: ChatStore, which is one fewer import and a simpler declaration

File: src/FormattedText.tsx [hunk 2]
```diff
@@ -8,15 +8,12 @@ interface Props {
...
-}
-
-export class FormattedText extends React.Component<Props, State> {
+export class FormattedText extends React.Component<Props, {}> {
     constructor(props: Props) {
         super(props);
     }
 
-    shouldComponentUpdate(nextProps: Props, nextState: State, nextContext: any): boolean {
+    shouldComponentUpdate(nextProps: Props, nextState: {}, nextContext: any): boolean {
```
Comment (billba) [inline src/FormattedText.tsx]: cleaner to just not list the last two parameters since you don't use them.

File: src/Store.ts (hunk 1 above)
Comment (eanders-MS) [inline src/Store.ts]: OK, I've made this change.

File: src/FormattedText.tsx (hunk 2 above)
Comment (eanders-MS) [inline src/FormattedText.tsx]: Ok, I've made this change.

**Pull request merged**